README.md

```diff
@@ -3,49 +3,60 @@
 [![PyPI](https://img.shields.io/pypi/v/truststore)](https://pypi.org/project/truststore)
 [![CI](https://github.com/sethmlarson/truststore/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/sethmlarson/truststore/actions/workflows/ci.yml)
 
-Verify certificates using OS trust stores. Supports macOS, Windows, and Linux (with OpenSSL). **This project should be considered experimental.**
+Verify certificates using OS trust stores. This is useful when your system contains
+custom certificate authorities such as when using a corporate proxy or using test certificates.
+Supports macOS, Windows, and Linux (with OpenSSL).
 
-## Usage
+## Installation
 
-```python
-# The following code works on Linux, macOS, and Windows without dependencies.
-import socket
-import ssl
-import truststore
+Truststore is installed from [PyPI](https://pypi.org/project/truststore) with pip:
 
-# Create an SSLContext for the system trust store
-ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+```{code-block} shell
+$ python -m pip install truststore
+```
 
-# Connect to the peer and initiate a TLS handshake
-sock = socket.create_connection(("example.com", 443))
-sock = ctx.wrap_socket(sock, server_hostname="example.com")
+Truststore **requires Python 3.10 or later** and supports the following platforms:
+- macOS 10.8+ via [Security framework](https://developer.apple.com/documentation/security)
+- Windows via [CryptoAPI](https://docs.microsoft.com/en-us/windows/win32/seccrypto/cryptography-functions#certificate-verification-functions)
+- Linux via OpenSSL
 
-# Also works with libraries that accept an SSLContext object
-import urllib3
+## User Guide
 
-http = urllib3.PoolManager(ssl_context=ctx)
-http.request("GET", "https://example.com")
+You can inject `truststore` into the standard library `ssl` module so the functionality is used
+by every library by default. To do so use the `truststore.inject_into_ssl()` function:
```

Comment:
@sethmlarson We should mention the importance of doing this as early as possible / the caveat that the injection won't impact modules that already imported SSLContext.

Reply:
That's a good idea, thank you!